src/servers/ZoneServer2016/entities/lootableconstructionentity.ts

```diff
@@ -352,4 +355,46 @@ export class LootableConstructionEntity extends BaseLootableEntity {
     const damage = damageInfo.damage * (3 * freePlaceDmgMultiplier);
     this.damage(server, { ...damageInfo, damage });
   }
+
+  OnExplosiveHit(
+    server: ZoneServer2016,
+    sourceEntity: BaseEntity,
+    client?: ZoneClient2016
+  ) {
+    if (!isPosInRadius(2, this.state.position, sourceEntity.state.position))
```

Comment:
True

Reply:
Actually since different entities have different radius logic we can't really have a radius as a field and have it be consistent across the board, we'd have to redo explosive distance calculations so that every entity can use just a single radius